client/components/Assignments/AssignmentItem/fields/Content.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {assignmentUtils} from '../../../../utils';
+import {Label} from '../../..';
+
+interface IProps {
+    assignment: any;
+}
+
+export const ContentComponent = ({assignment}: IProps) => {
+    const hasContent = assignmentUtils.assignmentHasContent(assignment);
+
+    if (!hasContent) {
+        return <span />;
```

Comment:
Is returning `<span />` here achieving anything, or can `null` be returned instead?

Reply:
Yeah I meant to put null